src/app/work-item/work-item-list/work-item-list-entry/work-item-list-entry.component.scss

```diff
@@ -1,16 +1,31 @@
 @import "./../../../../assets/stylesheets/custom";
 
-
 .list-group-item {
   cursor: pointer;
   font-size: rem(12);
   align-items: center;
   .row-cbh {
     margin-right: em(10);
     margin-left: em(5);
-    padding-right: em(10);
-    border-right: 1px solid $color-pf-black-500;
+    padding:em(2) em(10) em(2) 0;
+    border-right: 1px solid $color-pf-white ;
+    input[type=checkbox] {
+      margin: 0;
```

Comment:
code from line 77 is not required. It got duplicated. margin 0 we need to overwrite 5px margin top